langchain4j-anthropic/src/test/java/dev/langchain4j/model/anthropic/internal/sanitizer/MessageSanitizerTest.java

```diff
@@ -0,0 +1,141 @@
+package dev.langchain4j.model.anthropic.internal.sanitizer;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.data.message.SystemMessage;
+import dev.langchain4j.data.message.UserMessage;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.List;
+import dev.langchain4j.data.message.AiMessage;
+
+class MessageSanitizerTest {
+
+    @Test
+    void test_stripSystemMessage() {
+        List<ChatMessage> messages = new ArrayList<>();
+        messages.add(new SystemMessage("System message"));
+        messages.add(new UserMessage("User message"));
+
+        List<ChatMessage> sanitized = MessageSanitizer.sanitizeMessages(messages);
+
+        assertEquals(1, sanitized.size());
+        assertTrue(sanitized.get(0) instanceof UserMessage);
+    }
+
+    @Test
+    void test_stripMultipleSystemMessages() {
+        List<ChatMessage> messages = new ArrayList<>();
+        messages.add(new SystemMessage("System message 1"));
+        messages.add(new SystemMessage("System message 2"));
+        messages.add(new UserMessage("User message"));
+
+        List<ChatMessage> sanitized = MessageSanitizer.sanitizeMessages(messages);
+
+        assertEquals(1, sanitized.size());
+        assertTrue(sanitized.get(0) instanceof UserMessage);
```

Comment:
could you please also check which exactly user message is left? in other tests as well